core/app/models/spree/import.rb

```
@@ -0,0 +1,44 @@
+
+      def type_for_model(model)
+        available_types.find { |type| type.model_class.to_s == model.to_s }
+      end
+
+      # eg. Spree::ImportService::Products => Spree::Product
+      def model_class
+        "Spree::#{to_s.demodulize.singularize}".constantize
+      end
+    end
+  end
+end
```

RuboCop (1.75.5)

[convention] Final newline missing.

(Layout/TrailingEmptyLines)